src/app/components/Shared/button.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+export default function Button({ label }) {
```

Comment:
Why nest it in a `Shared` folder. From my understanding `src/app/components` is for shared components. So no need to nest it even deeper.